### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by yang1666204**

### CODE REVIEW

File: ui/src/components/customModal/MyInfoModal.tsx [hunk 1]
```diff
@@ -13,32 +14,48 @@ export default function MyInfoModal({
...
+        defaultMessage: '描述',
+      }),
       children: `${accountInfo?.description || '-'}`,
     },
     {
       key: 'lastLoginAt',
-      label: '最近一次登录',
+      label: intl.formatMessage({
+        id: 'src.components.customModal.685C8A0E',
+        defaultMessage: '最近一次登录',
```
Comment (powerfooI) [inline ui/src/components/customModal/MyInfoModal.tsx:53]: Modify the label to `最后一次登录时间`

File: ui/src/pages/Access/Accounts.tsx [hunk 2]
```diff
@@ -71,21 +86,30 @@ export default function Accounts({
...
       dataIndex: 'roles',
       render: (roles: AcRole[]) => (
         <span>{roles.map((role) => role.name).join(',')}</span>
       ),
     },
     {
-      title: '最近一次登陆时间',
+      title: intl.formatMessage({
+        id: 'src.pages.Access.013B4789',
+        defaultMessage: '最近一次登陆时间',
```
Comment (powerfooI) [inline ui/src/pages/Access/Accounts.tsx:102]: `登录`

File: ui/src/pages/ResetPwd/index.tsx [hunk 3]
```diff
@@ -34,9 +35,14 @@ export default function ResetPwd() {
       </div>
       <Alert
         className={styles.alertContent}
-        message="您之前没有登录过，当前密码为默认密码长期使用不安全。请先修改密码再继续使用 oceanbase dashboard！"
+        message={intl.formatMessage({
+          id: 'src.pages.ResetPwd.2768C822',
+          defaultMessage:
+            '您之前没有登录过，当前密码为默认密码长期使用不安全。请先修改密码再继续使用 oceanbase dashboard！',
```
Comment (powerfooI) [inline ui/src/pages/ResetPwd/index.tsx:41]: Add a comma at `默认密码，长期使用不安全`

File: ui/src/i18n/strings/zh-CN.json [hunk 4]
```diff
@@ -1040,5 +1040,101 @@
...
+  "src.pages.Access.4C7AA3C7": "操作",
+  "src.pages.Access.B6BC9A53": "重置密码",
+  "src.pages.Access.5339CE19": "编辑",
+  "src.pages.Access.68661542": "你确定要删除该用户吗？",
+  "src.pages.Access.C77EB99D": "删除",
+  "src.components.customModal.0D1428CF": "操作成功！",
+  "src.components.customModal.BC49A0BD": "修改密码",
+  "src.components.customModal.D33CD1F7": "请输入",
+  "src.components.customModal.ADFC0106": "原密码",
+  "src.components.customModal.E97DEF21": "请输入",
```
Comment (powerfooI) [inline ui/src/i18n/strings/zh-CN.json:1086]: Why do we need so many items with the same value? Can we merge them?

File: ui/src/components/customModal/HandleAccountModal.tsx [hunk 5]
```diff
@@ -81,66 +104,159 @@ export default function HandleAccountModal({
...
+              id: 'src.components.customModal.94E51D06',
+              defaultMessage: '用户名',
+            })}
             name={'username'}
           >
-            <Input placeholder="请输入" />
+            <Input
+              placeholder={intl.formatMessage({
+                id: 'src.components.customModal.05BFF296',
+                defaultMessage: '请输入',
```
[the remark this answers is not in the document]
Comment (yang1666204) [reply ui/src/components/customModal/HandleAccountModal.tsx:125]: We use [oic](https://www.npmjs.com/package/@oceanbase-odc/ob-intl-cli) as an `internationalization tool`, but it does not support merging identical items. In fact, I recently planned to participate in the construction of oic to support some practical problems encountered in the project.

**Review by powerfooI — APPROVED** [review]

**Pull request merged**